vite.config.ts

```diff
@@ -30,14 +30,15 @@ export default defineConfig({
       hook: 'writeBundle',
       targets: Object.entries({
         textLayer: [
-          [1323, 1348],
-          [2543, 2553],
+          [1323, 1343],
+          [1347, 1348],
+          [2549, 2559],
           [231, 349],
-          [1362, 1373],
         ],
         annotationLayer: [
-          [1323, 1348],
-          [2543, 2553],
+          [1323, 1343],
+          [1347, 1348],
+          [2549, 2559],
           [350, 712],
         ],
```

Comment:
I think it should be like this (please correct me if I'm wrong):

```suggestion
        textLayer: [
          [1323, 1343],
          [1347, 1348],
          [2546, 2555],
          [231, 349],
        ],
        annotationLayer: [
          [1323, 1343],
          [1347, 1348],
          [2546, 2555],
          [350, 712],
        ],
```

Reply:
You are right. I was three lines off the mark.
I noticed, that all remaining css variables in the `:root` selector weren't used anywhere in the css files. So I excluded them also.

Removed lines:
```css
:root {
  --outline-width: 2px;
  --outline-color: #0060df;
  --outline-around-width: 1px;
  --outline-around-color: #f0f0f4;
  --hover-outline-around-color: var(--outline-around-color);
  --focus-outline: solid var(--outline-width) var(--outline-color);
  --unfocus-outline: solid var(--outline-width) transparent;
  --focus-outline-around: solid var(--outline-around-width) var(--outline-around-color);
  --hover-outline-color: #8f8f9d;
  --hover-outline: solid var(--outline-width) var(--hover-outline-color);
  --hover-outline-around: solid var(--outline-around-width) var(--hover-outline-around-color);
  --freetext-line-height: 1.35;
  --freetext-padding: 2px;
  --resizer-bg-color: var(--outline-color);
  --resizer-size: 6px;
  --resizer-shift: calc(0px - (var(--outline-width) + var(--resizer-size)) / 2 - var(--outline-around-width))
}
```

That simplifies the css ranges to:
```ts
textLayer: [
  [2546, 2555],
  [231, 349],
],
annotationLayer: [
  [2546, 2555],
  [350, 712],
],
```

Reply:
Indeed, thanks!